skfda/_utils/_utils.py

```diff
@@ -6,6 +6,37 @@
 
 import numpy as np
 
+def _check_univariate(fd):
+    """Checks if an FData is univariate and raises an error"""
+
+    if fd.dim_domain != 1 or fd.dim_codomain != 1:
+        raise ValueError(f"The functional data must be univariate, i.e.,"
+                         f"with dim_domain=1 ({fd.dim_domain}) and "
```

Comment:
I think it could be made clearer that the value inside parentheses is the actual value.

Reply:
Change. Now the message is something like
> ValueError: The functional data must be univariate, i.e., with dim_domain=1 (currently is 2) and dim_codomain=1